apps/backend/lib/src/pagination.ts

```diff
@@ -0,0 +1,15 @@
+import { Request, Response, NextFunction } from "express";
+
+// Pagination middleware. Just ensures that the two pagination parameters are on
+// the request body so you don't need to error check for them in every route.
+export const usePagination = (
+  req: Request,
+  res: Response,
+  next: NextFunction
+) => {
+  const num_items = parseInt(req.query.num_items as string) || 10;
+  const offset = parseInt(req.query.offset as string) || 0;
```

Comment:
Can this throw? If so, let's make sure that our error catching Middleware is in front of this so we don't accidently leak stack traces

Reply:
Nope, if `parseInt` can't parse out an int, it returns a `NaN` which is falsy, so the expression would resolve to `10` (or `0`).